diff -aurN gnutls-3.1.3.orig/configure.ac gnutls-3.1.3/configure.ac
--- gnutls-3.1.3.orig/configure.ac	2012-10-18 22:52:15.790921079 +0200
+++ gnutls-3.1.3/configure.ac	2012-10-18 22:56:23.630096392 +0200
@@ -25,7 +25,7 @@
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([1.11.3 no-dist-gzip dist-xz dist-lzip -Wall -Werror -Wno-override])
+AM_INIT_AUTOMAKE([1.11.3 no-dist-gzip dist-xz dist-lzip -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AM_CONFIG_HEADER(config.h)
 
@@ -37,6 +37,8 @@
 AC_PROG_CC
 AM_PROG_AS
 AC_PROG_CXX
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
 gl_EARLY
 
 # For includes/gnutls/gnutls.h.in.
